assets/src/edit-story/elements/text/karma/edit.karma.js

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { Editor, EditorState } from 'draft-js';
+import { fireEvent } from '@testing-library/react';
+
+/**
+ * Internal dependencies
+ */
+import { Fixture } from '../../../karma/fixture';
+import { useStory } from '../../../app/story';
+import { useInsertElement } from '../../../components/canvas';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../../app/font/defaultFonts';
+import { getSelectionForAll } from '../../../components/richText/util';
+
+describe('TextEdit integration', () => {
```

Comment:
+1 for marking tests as 'integration', so that there's a clear separation of test types.

Reply:
We can also make this `describe.integration(...)` to make it even more clear and we'd pass `fixture` as an argument or a global. 

Also this is possible: we may decide in the future to completely drop Jest tests. The main difference between unit and integration test in this case is style - how much of the application we're rendering, mocking, etc. I'm certain that we will otherwise end up with a technology that's completely compatible between unit and integration tests. I'd still keep files clearly labels in folders or name to make it clear. But if Karma works well for us, it will be a question why not just use it for unit tests as well. Karma also shows to be ~2-3x faster in my tests.